hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -525,10 +518,7 @@ private static void invokeSetStoragePolicy(final FileSystem fs, final Path path,
 
     try {
       fs.setStoragePolicy(path, storagePolicy);
-
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("Set storagePolicy={} for path={}", storagePolicy, path);
-      }
+      LOG.debug("Set storagePolicy={} for path={}", storagePolicy, path);
```

Comment:
nit: `path` is not an instance of `String`, so auto-type-conversion means this will call `path.toString()` even when debug is not enabled. That method looks entirely too complicated, with lots of branching, but I guess nothing too hateful. Just and FYI.